main/ui/src/main/java/org/cryptomator/ui/unlock/UnlockWorkflow.java

```diff
@@ -129,8 +129,12 @@ private void handleSuccess() {
 				window.setScene(successScene.get());
 				window.show();
 			});
-			case REVEAL -> vaultService.reveal(vault);
-			case IGNORE -> {}
+			case REVEAL -> {
+				Platform.runLater(window::close);
+
```

Comment:
remove blank line, just for aesthetics, since both lines are tightly related to the `REVEAL` case and we're not talking about two separate coherent blocks of logic here.